src/Scripts/popup.js

```diff
@@ -524,34 +524,54 @@ const renderSecuritySummaryOSSIndex = (message) => {
 const renderLicenseData = (message) => {
   var thisComponent = message.message.response.componentDetails["0"];
   let licenseData = thisComponent.licenseData;
+
+  // Temporary fix, HTML, etc.. should likely be revamped
   if (licenseData.declaredLicenses.length > 0) {
-    if (licenseData.declaredLicenses["0"].licenseId) {
-      //$("#declaredlicenses_licenseId").html(licenseData.declaredLicenses["0"].licenseId);
-      //<a href="https://en.wikipedia.org/wiki/{{{licenseId}}}_License" target="_blank">https://en.wikipedia.org/wiki/{{{licenseId}}}_License</a>
-      //https://spdx.org/licenses/0BSD.html
+    for (let i = 0; i < licenseData.declaredLicenses.length; i++) {
       let link =
         "https://en.wikipedia.org/wiki/" +
-        licenseData.declaredLicenses["0"].licenseId +
+        licenseData.declaredLicenses[i].licenseId +
         "_License";
-      console.log("link", link, licenseData.declaredLicenses["0"].licenseName);
-      $("#declaredlicenses_licenseLink").attr("href", link);
-      $("#declaredlicenses_licenseLink").html(
-        licenseData.declaredLicenses["0"].licenseId
+
+      $('<a>', {
+        text: licenseData.declaredLicenses[i].licenseId,
+        title: 'License ID',
+        href: link,
+        target: '_blank',
+      }).appendTo("#declaredLicensesSpan");
+
+      if (licenseData.declaredLicenses.length - i != 1) {
+        $("#declaredLicensesSpan").append(", ");
```

Comment:
@DarthHater Won't this mean that when there are more than 1 links that a comma is appended to the LAST one in the list as well?